### PR TITLE
Load nksip in parallel with big timeouts

### PR DESCRIPTION
**Pull request by NelsonVides**

### CODE REVIEW

**Comment by mongoose-im** [comment]
[8734.1](https://travis-ci.org/esl/MongooseIM/jobs/744388317) / Erlang 23.0.3 / small_tests / 28b072984b1c813faf5197882cc9f0cab472fc9e
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2947/8734/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2947/8734/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2020-11-18_11.03.11/lib.mongooseim.logs/run.2020-11-18_11.03.11/suite.log.html)

File: big_tests/tests/jingle_SUITE.erl [hunk 1]
```diff
@@ -78,6 +69,46 @@ init_per_suite(Config) ->
...
+             fun() ->
+                     {ok, _} = rpc(RPCSpec1#{timeout => Timeout}, gen_mod, start_module,
+                                   [
+                                    Host, mod_jingle_sip,
+                                    [{proxy_host, "localhost"},
+                                     {proxy_port, Port},
+                                     {username_to_phone,[{<<"2000006168">>, <<"+919177074440">>}]}]
+                                   ])
+             end),
+    Pid2 = proc_lib:spawn_link(
```
Comment (chrzaszcz) [inline big_tests/tests/jingle_SUITE.erl:89]: Could we have a helper function here and call it with args like (RPCSpec, ExtraOpts)?

```
start_nskip(mim(), []),
start_nksip(mim2(), [{listen_port, 12346}]),
```

**Review by chrzaszcz — COMMENTED** [review]
Just a comment from me.
Another thing: could we revert the original timeout for starting modules? It was changed only because of nksip, right?

**Comment by mongoose-im** [comment]
[8738.1](https://travis-ci.org/esl/MongooseIM/jobs/744883751) / Erlang 23.0.3 / small_tests / 7f2c9cf4058827a437db8ea08c8b7bfd27ba1b12
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2947/8738/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2947/8738/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2020-11-20_14.29.14/lib.mongooseim.logs/run.2020-11-20_14.29.14/suite.log.html)

**Comment by mongoose-im** [comment]
[8740.1](https://travis-ci.org/esl/MongooseIM/jobs/744964807) / Erlang 23.0.3 / small_tests / 72e9edbfbdecaa27697fa6393cbf072395d88332
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2947/8740/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2947/8740/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2020-11-21_00.48.17/lib.mongooseim.logs/run.2020-11-21_00.48.17/suite.log.html)

**Review by chrzaszcz — APPROVED** [review]
Looks good, thanks for the improvements!

**Pull request merged**